maven-surefire-plugin/src/site/apt/examples/junit-platform.apt.vm

```diff
@@ -0,0 +1,206 @@
+ ------
+ Using JUnit Platform
+ ------
+ JUnit Lambda Team <junit-lambda-team@googlegroups.com>
+ ------
+ 2018-05-14
+ ------
+ 
+ ~~ Licensed to the Apache Software Foundation (ASF) under one
+ ~~ or more contributor license agreements.  See the NOTICE file
+ ~~ distributed with this work for additional information
+ ~~ regarding copyright ownership.  The ASF licenses this file
+ ~~ to you under the Apache License, Version 2.0 (the
+ ~~ "License"); you may not use this file except in compliance
+ ~~ with the License.  You may obtain a copy of the License at
+ ~~
+ ~~   http://www.apache.org/licenses/LICENSE-2.0
+ ~~
+ ~~ Unless required by applicable law or agreed to in writing,
+ ~~ software distributed under the License is distributed on an
+ ~~ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ ~~ KIND, either express or implied.  See the License for the
+ ~~ specific language governing permissions and limitations
+ ~~ under the License.
+
+ ~~ NOTE: For help with the syntax of this file, see:
+ ~~ http://maven.apache.org/doxia/references/apt-format.html 
+
+Using JUnit Platform
+
+* Configuring JUnit Platform
+
+  To get started with JUnit Platform, you need to add at least a single <<<TestEngine>>> implementation
+  to your project. For example, if you want to write tests with Jupiter, you must add the
+  <<<junit-jupiter-engine>>> to the dependencies like:
+
++---+
+<dependencies>
+  [...]
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <version>5.2.0</version>
+      <scope>test</scope>
+    </dependency>
+  [...]
+</dependencies>
++---+
+
+  This will pull in all required dependencies. Among those dependencies is <<<junit-jupiter-api>>> which contains
+  the classes and interfaces your test source requires to compile. <<<junit-platform-engine>>> is also resolved and
+  added.
+
+  This is the only step that is required to get started - you can now create tests in your test source directory
+  (e.g., <<<src/test/java>>>).
+
+  If you want to write and execute JUnit 3 or 4 tests via the JUnit Platform add the Vintage Engine to your projects'
+  dependencies:
+
++---+
+<dependencies>
+  [...]
+    <dependency>
+      <groupId>org.junit.vintage</groupId>
+      <artifactId>junit-vintage-engine</artifactId>
+      <version>5.2.0</version>
+      <scope>test</scope>
+    </dependency>
+  [...]
+</dependencies>
++---+
+
+* Provider Selection
+
+   If nothing is configured, Surefire detects which JUnit version to use by the following algorithm:
+
++---+
+if the JUnit Platform Engine is present in the project
+    use junit-platform
+if the JUnit version in the project >= 4.7 and the parallel attribute has ANY value
```

Comment:
Pls emphase with `<<<parallel>>` but the attribute should be rewritten to `configuration parameter`.

Reply:
Sure.